13724 console parallelization (#13790)
pimcore/pimcore: doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

- [PhpArrayTable]: Removed PhpArrayTable class
- [Elements] Changed method signature on `Pimcore\Model\Element\ElementInterface::save()`, this changes the `::save()` method on all classes (e.g. DataObjects and Pages) implementing the interface, including those inheriting from `Concrete`/`AbstractObject`, see [#13207](https://github.com/pimcore/pimcore/issues/13207)
- [Document Editables] Removed method_exists bc layer for `getDataEditmode()`, `rewriteIds()` & `load()`, please use the corresponding interfaces `EditmodeDataInterface`, `IdRewriterInterface` & `LazyLoadingInterface` instead.

> - [Parallelization] 

Reviewer: ```suggestion
- [Commands] 
```

>   - Removed `webmozarts/console-parallelization` dependency due to maintainability reasons. If you still want to use parallelization for console commands, please add the dependency to your own `composer.json`.

Reviewer: ```suggestion
  - Removed `webmozarts/console-parallelization` dependency to make parallelization optional. If you still want to use parallelization for console commands, please add the dependency to your own `composer.json`.
```

>   - Removing the `configureParallelization()` method from the trait. Please call `$this->configureCommand()` instead to use the standard `webmozarts/console-parallelization` functionality

Reviewer: ```suggestion
  - Removed the deprecated `Parallelization::configureParallelization()` method.
```
New method should be added on 10.6 with deprecation notes.
